tests/test_event_listener.rs

```diff
@@ -85,6 +85,63 @@ impl EventListener for EventCounter {
     }
 }
 
+#[derive(Default, Clone)]
+struct StallEventCounter {
+    flush: Arc<AtomicUsize>,
+    stall_conditions_changed_num: Arc<AtomicUsize>,
+    triggered_writes_slowdown: Arc<AtomicUsize>,
+    triggered_writes_stop: Arc<AtomicUsize>,
+}
+
+impl EventListener for StallEventCounter {
+    fn on_flush_completed(&self, info: &FlushJobInfo) {
+        assert!(!info.cf_name().is_empty());
+        self.flush.fetch_add(1, Ordering::SeqCst);
+        self.triggered_writes_slowdown
+            .fetch_add(info.triggered_writes_slowdown() as usize, Ordering::SeqCst);
+        self.triggered_writes_stop
+            .fetch_add(info.triggered_writes_stop() as usize, Ordering::SeqCst);
+    }
+
+    fn on_stall_conditions_changed(&self, info: &WriteStallInfo) {
+        assert!(!info.cf_name().is_empty());
```

Comment:
Can we assert something like 
```
assert_eq!(info.cf_name(), expected_name);
```

Reply:
Created with default config, names of those ColumnFamily are all `default`.

Reply:
@DorianZheng I have updated the test by creating and using another cf.